electricitymap/contrib/capacity_parsers/REE.py

```diff
@@ -60,9 +60,9 @@ def fetch_production_capacity(
     geo_ids = GEO_LIMIT_TO_GEO_IDS[geo_limit]
     url = "https://apidatos.ree.es/es/datos/generacion/potencia-instalada"
     params = {
-        "start_date": target_datetime.strftime("%Y-01-01T00:00"),
-        "end_date": target_datetime.strftime("%Y-12-31T23:59"),
-        "time_trunc": "year",
+        "start_date": target_datetime.strftime("%Y-%m-01T00:00"),
+        "end_date": target_datetime.strftime("%Y-%m-31T23:59"),
```

Comment:
it seems like the code works if running for November but this can lead to errors for months that do not have 31 days.

Reply:
This should do the trick:
```python
calendar.monthrange(test_date.year, test_date.month)[1]
```

Reply:
It works for me for other months, even February. I could change it to"%Y-%m-01T23:59", if you wish. That one seems to also work. Is it necessary to be the last day of the month?

Reply:
no i don't think it need to be the last day if the parser returns the same data when you query for 1 day